### ISSUE TITLE
confusion about the regularization term in formular (11)

### ISSUE THREAD

**Issue by Terry10086** [LeapLabTHU/ActiveNeRF#7]
Thank you for your impressive work! I have a couple of questions regarding your Equation 11 and its corresponding code snippet:
Equation 11 corresponds to the following code
`img2mse_uncert_alpha = lambda x, y, uncert, alpha, w : torch.mean((1 / (2*(uncert+1e-9).unsqueeze(-1))) *((x - y) ** 2)) + 0.5*torch.mean(torch.log(uncert+1e-9)) + w * alpha.mean() + 4.0
`
My main query is regarding the difference between the regularization term in the formula. The regularization term in formula (11) uses **the volume density** , while the code segment uses **alpha.mean()**. Could you please clarify why the volume density isn't utilized in the code? Additionally, I'm curious about the significance of adding 4.0 at the end of the loss function. 
Why not use the volume density and why plus 4 at the end of the loss function?
